docs/04-resources/04-tutorials/rootstock-vyper.md

```diff
@@ -0,0 +1,222 @@
+---
+sidebar_label: Vyper Smart Contract on RootStock
+sidebar_position: 7
+title: Deploying a Vyper Smart Contract to RootStock (RSK) Testnet using Python
+description: "This guide walks through the process of deploying a smart contract to the RootStock (RSK) testnet using Python and Web3.py. We'll be deploying a simple Vyper contract that demonstrates how to interact with the RSK network."
+tags: 
+  - rsk
+  - rootstock
+  - tutorials
+  - resources
+  - tests
+  - web3py
+  - vyper
+  - smart contracts
+  - python
+  - developers
+---
+
+This guide demonstrates how to deploy smart contracts written in Vyper to the RootStock (RSK) testnet using Python and Web3.py. RSK is a groundbreaking smart contract platform that's merge-mined with Bitcoin, offering unique advantages for developers:
+
+- **Bitcoin Compatibility**: Deploy smart contracts while leveraging Bitcoin's security and network effects
+- **EVM Compatibility**: Use familiar Ethereum tools and practices while building on Bitcoin
+- **Lower Fees**: Benefit from RSK's cost-effective transaction fees
+- **Scalability**: Enjoy higher transaction throughput compared to the Bitcoin mainnet
+
+We'll walk through creating a simple Vyper contract and deploying it to RSK's testnet, covering everything from environment setup to handling RSK-specific configurations. Whether you're an experienced Ethereum developer looking to expand to Bitcoin-based smart contracts, or just starting your blockchain journey, this guide will help you get up and running with RSK.
+
+## Prerequisites
+
+- [uv](https://docs.astral.sh/uv/)
+  - You'll know you've done it right if you can run `uv --version` and see a version number.
+- [git](https://git-scm.com/)
+  - You'll know you've done it right if you can run `git --version` and see a version number.
+  - Helpful shortcut:
+
+```bash
+# For bash
+echo "source $HOME/.bashrc >> $HOME/.bash_profile"
+
+# For zsh
+echo "source $HOME/.zshenv >> $HOME/.zprofile"
+```
+
+- Python 3.x
+- A text editor
+- Basic understanding of smart contracts and Python
+- RSK testnet RBTC (will show you how to get this)
+
+## Installation
+
+```bash
+git clone https://github.com/EdwinLiavaa/Web3py-Vyper-RootStock.git
+cd Web3py-Vyper-RootStock
+```
+
+### uv
+
+```bash
+uv sync
+```
+
+### pip/python
+
+```bash
+python -m venv ./venv
```

Comment:
This throws an error:

`python -m venv ./venv`

owanate@Owanates-MacBook-Pro Web3py-Vyper-RootStock % python -m venv ./venv
zsh: command not found: python

This probably happened because of my comment above.

Reply:
It looks like Python's venv module is not installed on your system. You are using a MacBook, so You can try fix this by:

Reinstalling Python and make sure to check the option to install pip and venv during installation
Additionally, you should ensure you have Python 3 installed and are using the correct Python command (sometimes it needs to be python3 instead of python).